### PR TITLE
test whether output file was actually written, #25

### PR DESCRIPTION
**Pull request by kba**

### CODE REVIEW

File: ocrd-fileformat-transform [hunk 1]
```diff
@@ -95,6 +95,11 @@ main () {
             continue
         fi
 
+        if [ -e "$out_file" ];then
```
Comment (mikegerber) [inline ocrd-fileformat-transform:98]: I think that's the other way around? This tests for existence, did you mean to negate that?

**Review by mikegerber — CHANGES_REQUESTED** [review]

File: ocrd-fileformat-transform (hunk 1 above)
Comment (kba) [reply ocrd-fileformat-transform:98]: oh, yes of course. 😊

Comment (kba) [reply ocrd-fileformat-transform]: 55f6b37

**Review by mikegerber — APPROVED** [review]
Now it looks good to me!

**Pull request merged**